job_dsl/rpc_gating_merge_trigger.groovy

```diff
@@ -0,0 +1,78 @@
+// Can't use env.FOO = {FOO} to transfer JJB vars to groovy
+// as this file won't be templated by JJB.
+// Alternative is to use parameters with JJB vars as the defaults.
+library "rpc-gating@${RPC_GATING_BRANCH}"
+common.globalWraps(){
+
+  // We do not want to trigger any of these
+  // for rpc-gating forks or branches other
+  // than 'master'. So if this is triggered
+  // by anything else, end now.
+  // TODO(odyssey4me):
+  // Remove once RE-1645 is resolved.
+  if (env.RPC_GATING_BRANCH != "master") {
```

Comment:
This is a funny one. Given that this is terminating the build right at the start, it essentially negates the ability to override the branch. The only reason the branch wouldn't be master is if someone is doing testing, in most situations that would automatically require this to be commented out. I guess it acts as extra protection against someone doing something stupid.